features/google/src/context/google-context.jsx

```diff
@@ -14,18 +16,30 @@ function updatePodNavigation(pod, history, handleBack, location) {
         : pod.polyNav.setActiveActions([]);
 }
 
+function updateTitle(pod, location, popUp) {
+    console.log(location.pathname.substring(1));
+    console.log(popUp);
+    pod.polyNav.setTitle(
+        location.pathname === "/"
+            ? ""
+            : popUp
+            ? i18n.t("navbarTitles:info")
+            : i18n.t(`navbarTitles:${location.pathname.substring(1)}`)
```

Comment:
How about a fallback in case there's no title for a screen? That way you wouldn't have to set `Google Data Importer` multiple times, and we also wouldn't crash if a new screen is added without a dedicated title.